lib/osf-components/addon/components/addons-service/addon-account-setup/component.ts

```diff
@@ -114,15 +127,13 @@ export default class AddonAccountSetupComponent extends Component<Args> {
                     labelText: t('addons.accountCreate.username-label'),
                     inputType: 'text',
                     inputPlaceholder: t('addons.accountCreate.username-placeholder'),
-                    inputValue: credentials.username,
```

Comment:
There is a `@value` binding to this `inputValue` on the template file (on line 123). It seems that the binding is not necessary in the first place since we programmatically update `this.credentialsObject` in the `inputFieldChange` action. With this key in the object gone, we should also remove that `@value` binding.